### ISSUE TITLE
There should be a way to run modules for single countries

### ISSUE THREAD

**Issue by chrmongeau** [SWS-Methodology/faoswsTrade#129]
Let's say there is a revision of the data for all years for a specific country. With the current design of modules, all countries need to be updated and the process requires (long) time. In other words, there is not a way to run a module for a specific country. We should think about implementing a way to run the module for a single country.

**Comment by malexan** [comment]
The question is in additional filter during raw trade data reading and in games with area codes, right? 

BTW, currently there are two additional parameters in the `main.R`: logical `stopaftermapping` and numeric `samplesize` for ES data (if it is NULL, it is not applied - default) - should be extended to TL as well.

**Comment by chrmongeau** [comment]
> The question is in additional filter during raw trade data reading [...]

Yes, though I think that just filtering is not sufficient (e.g., the remaining countries would be considered as nonreporters, but in this case we don't really care about "nonreporters").

> [...] and in games with area codes, right?

Mmh, I don't fully understand this :/

> `stopaftermapping` and `samplesize` [...] should be extended to TL as well.

Just `samplesize` should be used also in TL, right?

Oh, about other parameters: [`debughsfclmap`](https://github.com/SWS-Methodology/faoswsTrade/blob/0bbd7a15e676da0b1f23e3938ae5efce024417fd/modules/complete_tf_cpc/main.R#L47) is used only in [vignettes/preanalysis_2009.Rmd](https://github.com/SWS-Methodology/faoswsTrade/blob/0bbd7a15e676da0b1f23e3938ae5efce024417fd/vignettes/preanalysis_2009.Rmd) and [vignettes/producingSet_new.Rmd](https://github.com/SWS-Methodology/faoswsTrade/blob/0bbd7a15e676da0b1f23e3938ae5efce024417fd/vignettes/producingSet_new.Rmd) (and related "producingSet" docs). I think we can get rid of it.

**Comment by malexan** [comment]
> area code games

When to specify the area to work on, we will use fcl code. Then the algorithm should detect is it comes from TL or ES, then to convert to the corresponding area classification (m49|geonom) and to extract from the source.

> Just samplesize should be used also in TL, right?

Yes

>debughsfclmap

So it is used! :) Yep, let's remove it.
